src/modules/blocks/ticket/container-content/capacity/template.js

```diff
@@ -99,8 +99,12 @@ class Capacity extends PureComponent {
 		};
 
 		const handleTempCapacityChange = ( e, max ) => {
-			if ( e.target.value === '' || e.target.value > 0 && e.target.value <= max ) {
-				onTempCapacityChange( e );
+			if ( e.target.value === '' || e.target.value > 0 ) {
+				if ( max === undefined ) {
+					onTempCapacityChange( e );
+				} else if ( e.target.value <= max ) {
+					onTempCapacityChange( e );
+				}
 			}
 		};
```

Comment:
I feel like this can be simplified a bit so it's easier to read:
`		const handleTempCapacityChange = ( e, max ) => {
			// If value is set to 0 or less, bail.
			if ( e.target.value !== '' && e.target.value < 1 ) {
				return;
			}
			// If value is greater than the maximum value, bail.
			if ( max !== undefined && e.target.value > max) {
				return;
			}
			onTempCapacityChange( e );
		};`